docs/source/indepth_tutorial/mpi-evaluator.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
<p>I suggest updating this in light of our mail discussion. Models that require write access to a working directory are unlikely to work. Also make clearer that this is an experimental new addition and that we solicit feedback.</p><p> </p><p>The code will work under Mac. Windows is a different matter because of the mess that is mpi on windows. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/308/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Updated it in quite clear text.